rmw_fastrtps_shared_cpp/test/test_guid_utils.cpp

```diff
@@ -25,15 +25,19 @@ static constexpr size_t byte_array_size =
   eprosima::fastrtps::rtps::GuidPrefix_t::size +
   eprosima::fastrtps::rtps::EntityId_t::size;
 
+
```

Comment:
Are the extra lines intentional? Here and line 40?

Reply:
Nope, that's just my Python linting reflexes (2 lines in between functions). Rolled back in https://github.com/ros2/rmw_fastrtps/pull/393/commits/31ee7146a6dbbf594841be7eb039cff4011b8ad3.